app/scripts/directives/logViewer.js

```diff
@@ -446,17 +446,38 @@ angular.module('openshiftConsole')
                   access_token: AuthService.UserStore().getToken()
                 });
 
-                $scope.$watchGroup(['context.project.metadata.name', 'options.container', 'name'], function() {
-                  angular.extend($scope, {
-                    // The archive URL violates angular's built in same origin policy.
-                    // Need to explicitly tell it to trust this location or it will throw errors.
-                    kibanaArchiveUrl: $sce.trustAsResourceUrl(logLinks.archiveUri({
-                                        namespace: $scope.context.project.metadata.name,
-                                        namespaceUid: $scope.context.project.metadata.uid,
-                                        podname: name,
-                                        containername: $scope.options.container,
-                                        backlink: URI.encode($window.location.href)
-                                      }, $filter('annotation')($scope.context.project,'loggingDataPrefix')))
+                // var ssarVersion = APIService.getPreferredVersion('selfsubjectaccessreviews');
+                var ssar = {
+                  apiVersion: 'authorization.k8s.io/v1',
+                  // apiService: APIService.toAPIVersion(ssarVersion)
+                  kind: 'SelfSubjectAccessReview',
+                  spec: {
+                    resourceAttributes: {
+                      resource: 'pods/log',
+                      namespace: 'openshift-logging',
+                      verb: 'view'
+                    }
+                  }
+                };
+                var promise = DataService.create('selfsubjectaccessreviews', null, ssar, {namespace: 'openshift-logging'});
```

Comment:
Make sure the API version in the request is the same as the endpoint you're calling. I believe it would be

```js
DataService.create({ group: 'authorization.k8s.io', version: 'v1', resource: 'selfsubjectaccessreviews'}, ...
```

Reply:
We should be caching this response somewhere so we only check it once per session. You probably want to move this into an Angular service.

Reply:
What if I remove role from a user? That could be a possible danger it that user is logged in a session and has dirty cash.

Reply:
Disregard, we'll do another round of auth on our side.